server/public/model/client4.go

```diff
@@ -4824,15 +4820,6 @@ func (c *Client4) TestEmail(ctx context.Context, config *Config) (*Response, err
 	return BuildResponse(r), nil
 }
 
-func (c *Client4) TestNotifications(ctx context.Context) (*Response, error) {
```

Comment:
Could you re-add this and `Client4.testNotificationRoute` above? I think they got removed by a bad merge